.gitignore

```diff
@@ -1,4 +1,3 @@
-/bin/*
```

Comment:
Why that change?

Reply:
Couldn't you commit this, then commit `bin/create_node_symlink.php`, and then roll-back the .gitignore?